### DISCUSSION TITLE
关于引用外部数据源

### DISCUSSION THREAD

**Discussion by unknowissue** [hhyo/Archery#1354] [Q&A]
请教一个问题，就是我们在做ci时，都是用mock模拟的数据，为啥不用真实的MySQL\Oracle啥的嘞，
或者能够公网能访问外部数据源的？
刚刚学github，想请教一下。

**Comment by LeoQuote** [comment]
当时加附带的服务很麻烦的，特别oracle 也很难搞起来。 mock 比较方便

**Reply by unknowissue** [reply]
嗷嗷，我以为是github有啥个数限制，，，，
写mock我有点懵逼，😭😭😭😭
另外这个ci有啥办法能再本地跑么，我提交次数很多，我就像在刷屏一样🥶🥶🥶🥶🥶🥶
显得我很笨比，
虽然确实是笨比
💩💩💩

**Reply by LeoQuote** [reply]
可以在本地跑的，需要一个mysql 和redis settings 里 sync 要设置为true，你跑试试遇到什么问题就解决就好了
